samples/04.0-file-transfer/integrationtest/build.gradle.kts

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
rename folder to `integration-tests`? (consistency with  `:extensions:data-plane:integration-tests`)